lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -5034,24 +5034,21 @@ LookupAllConformancesInContextRequest::evaluate(
   return IDC->getLocalConformances(ConformanceLookupKind::All);
 }
 
-void TypeChecker::checkConformancesInContext(DeclContext *dc,
-                                             IterableDeclContext *idc) {
+void TypeChecker::checkConformancesInContext(IterableDeclContext *idc) {
+  auto *const dc = idc->getAsGenericContext();
+
   // For anything imported from Clang, lazily check conformances.
   if (isa<ClangModuleUnit>(dc->getModuleScopeContext()))
     return;
 
   // Determine the access level of this conformance.
-  Decl *currentDecl = nullptr;
   AccessLevel defaultAccess;
```

Comment:
Let's just get rid of this braced block and compute `defaultAccess` below the conditional.

Reply:
Do you mean
```cpp
AccessLevel defaultAccess;
if (auto nominal = dc->getSelfNominalDecl()) {
  defaultAccess = nominal->getFormalAccess();
} else {
  return;
}
```
?

Reply:
```
const auto *nominal = dc->getSelfNominalTypeDecl();
if (!nominal)
  return;
auto defaultAccess = nominal->getFormalAccess();
```

Reply:
Oh, the conditional *return*